recipes-omnect/initrdscripts/omnect-os-initramfs/omnect-device-service-setup

```diff
@@ -19,47 +19,70 @@ get_id() {
     echo $line | awk -F: '{print $3}'
 }
 
+fsck_handling() {
+    local omnect_run_dir=$1
+    local omnect_device_service_user=$2
+    local omnect_device_service_group=$3
+    local labels=("boot" "cert" "data" "etc")
+
+    echo "{}" > ${omnect_run_dir}/fsck.json
+    for i in ${labels[@]}; do
+        local content="$(get_fsck_status_base64 ${i} | base64 -d -)"
+        if [ -n "${content}" ]; then
+            jq --arg i "${i}" --arg content "${content}" '{($i): ($content)}' ${omnect_run_dir}/fsck.json > ${omnect_run_dir}/fsck.json.tmp || return 1
+            mv ${omnect_run_dir}/fsck.json.tmp ${omnect_run_dir}/fsck.json
+            set_bootloader_env_var omnect_fsck_${i} ""
```

Comment:
do I miss something or does this actually only makes sense for variable omnect_fsck_boot?
but even if that's the case, it won't do any harm afaics.

Reply:
no, see difference between uboot and grub handling of saving the fsck status.
since it doesn't harm, i decided to do it in every case.